va/va.h

```diff
@@ -780,6 +780,18 @@ typedef enum
      * implementation, multiple frames encode/decode can improve HW concurrency
      */
     VAConfigAttribMultipleFrame         = 40,
+    /** \brief indicates the number of scalable pipes minus 1 that the underline encoder supports.
```

Comment:
from view of application , I could not understand salable pipes, could you change to another word or explain what's it

Reply:
@XinfengZhang could you please have a look at updated description?

Reply:
I agree with Carl that it's not clear what scalable pipes are. Better to explain it at first or use another term